packages/@aws-cdk/aws-autoscaling/lib/auto-scaling-group.ts

```diff
@@ -7,8 +7,9 @@ import sns = require('@aws-cdk/aws-sns');
 
 import {
   CfnAutoScalingRollingUpdate, Construct, Duration, Fn, IResource, Lazy, PhysicalName, Resource, Stack,
-  Tag, Token, withResolved
+  Tag, withResolved
 } from '@aws-cdk/core';
+import { stringifyNumber } from '@aws-cdk/core/lib/util';
```

Comment:
We usually try not to "deep import" private things. How about exporting this under the `core.Tokenization` class?

Reply:
sounds good to me - I've moved it